Directory.Build.props

```diff
@@ -9,6 +9,11 @@
     <Import Project="build\config.props" />
     <Import Project="build\common.project.props" />
   </ImportGroup>
+  <PropertyGroup>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Comment:
Don't set unconditional properties. Might not apply to this property, but overwrites can lead to issues.

The changes would also normally go to common.project.props.